std/algorithm/mutation.d

```diff
@@ -1787,13 +1788,143 @@ Returns:
     a range containing all of the elements of range with offset removed
  */
 Range remove
-(SwapStrategy s = SwapStrategy.stable, Range, Offset...)
+(SwapStrategy s = SwapStrategy.stable, Range, Offset ...)
 (Range range, Offset offset)
-if (s != SwapStrategy.stable
-    && isBidirectionalRange!Range
-    && hasLvalueElements!Range
-    && hasLength!Range
-    && Offset.length >= 1)
+if (Offset.length >= 1)
+{
+    static if (isNarrowString!Range)
+    {
+        static assert(isMutable!(typeof(range[0])),
+                "Elements must be mutable to remove");
+        static assert(s == SwapStrategy.stable,
+                "Only stable removing can be done for character arrays");
+        return removeStableString(range, offset);
+    }
+    else
+    {
+        static assert (isBidirectionalRange!Range,
```

Comment:
The CircleCI style checker won't allow this to be merged until you change this line and line 1807 so there's no space between the `assert` and the `(`.

Reply:
done